test/test_tags.rb

```diff
@@ -14,6 +14,9 @@ def create_post(content, override = {}, converter_class = Jekyll::Converters::Ma
     if override['read_posts']
       site.posts.docs.concat(PostReader.new(site).read_posts(''))
     end
+    if override['read_collections']
```

Comment:
@parkr the bot (and I believe Rubocop too) doesn't track line width on Jekyll/Jekyll for now because we have a bunch of lines way over that and it was impacting the score for something so trivial that we could fix over time and didn't impact the actual code or it's readability for the most part.